app.py

```
@@ -106,20 +124,26 @@ def update_frame(frame_index):
                 window.after(POPUP_DURATION * 1000, window.destroy)
+                logging.info(f"Image window will close after {POPUP_DURATION} seconds.")
+        except Exception as e:
+            logging.error(f"Error loading image sequence: {e}")
+            messagebox.showerror("Error", f"Failed to load image: {e}")
 
             # Define button actions
             def on_accept():
+                logging.info("User accepted to view the image.")
                 self.root.after(0, load_image)
 
             def on_exit():
```

Ruff

Missing return type annotation for private function `on_exit`

Add return type annotation: `None`

(ANN202)